lib/cookie/secureContext.ts

```diff
@@ -0,0 +1,140 @@
+import { URL } from 'url';
+import { isIP, isIPv4, isIPv6 } from 'net';
```

Comment:
These lines will need to be removed because `tough-cookie` is sometimes used in JavaScript runtimes other than Node.js we don't allow builtin modules to be imported. You can see this in the [CI run for this PR](https://github.com/salesforce/tough-cookie/actions/runs/13434219778/job/37563458160?pr=498) or if you run `npm run lint` locally. 

For the `URL` object, it doesn't need an import as it's already [available as a global](https://developer.mozilla.org/en-US/docs/Web/API/URL) in most JavaScript runtimes.

I'll outline some alternatives you can use instead of `isIP`, `isIPv4`, and `isIPv6` in the relevant areas below.